test/file.e2e-spec.ts

```diff
@@ -308,27 +308,27 @@ describe('File e2e', () => {
         page: 1,
       });
       expect(firstPage.total).toEqual(expectedTotalChildren);
-      expect(firstPage.hasMore).toBeTruthy();
+      expect(firstPage.hasMore).toBeFalsy();
```

Comment:
Other than the `beforeAll` -> `beforeEach`, nothing should be changed here.

Reply:
Actually `expectedTotalFiles` should include `expectedTotalVideos`. Sorry about that.